lib/release_notes_bot_web/controllers/webhook_controller.ex

```diff
@@ -29,7 +29,7 @@ defmodule ReleaseNotesBotWeb.WebhookController do
 
       _ ->
         # Check if repo exists.
-        {:ok, repo} = Repositories.find_or_create(body["repository"])
+        {:ok, repo} = Repositories.webhook_find_or_create(body["repository"])
```

Comment:
What about `find_or_create_by_{thing}` as a pattern? `find_or_create_by_webhook` in this case

Reply:
This seems like a great idea